### ISSUE TITLE
Las Vegas Building Data is missing

### ISSUE THREAD

**Issue by alperdincer** [SpaceNetChallenge/SpaceNetChallenge.github.io#63]
We are looking for a full set of SpaceNet Las Vegas Building vector data, but we only found the parts intersecting with training set and there are missing part of buildings. How can we get all the buildings in Las Vegas? 

Thanks.

**Comment by avanetten** [comment]
This is by design, as distributing the vector data for the test set would ruin testing.
